.github/workflows/simulation-1.2.yml

```diff
@@ -227,7 +227,7 @@ jobs:
       COVERAGE: 1
       MULTIPLY: 3
       PYTHONUNBUFFERED: 1
-      OTBR_COMMIT: "bf28f461fa2a8551f0a4161c04b87f7240856608" # Date:   Wed Oct 14 12:00:44 2020 +0800
+      OTBR_COMMIT: "d18242b0af81494e3a3c996cd1b2ab3b001f247c" # Date:   Mon Oct 26 15:27:04 2020 +0800
```

Comment:
the PR in OTBR is https://github.com/openthread/ot-br-posix/pull/598. The two PRs depend on each other. @jwhui should we merge this PR first or do the reverse?

Reply:
We can merge this PR first then update OTBR.

Reply:
Either way is fine for me. One of the repos will require two merges.

Reply:
Generally I prefer to merge OT PR first because in this way we can always merge PRs with all checks passed. We can't pass LGTM.com checks for OTBR if it's not using a commit of OpenThread master branch.

Reply:
@jwhui What about we merging this PR first?